src/configurations/destinations/fb_custom_audience/schema.json

```diff
@@ -21,232 +21,103 @@
         "pattern": "(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|^(.{0,100})$"
       },
       "userSchema": {
```

Comment:
I don't see change in db-config.json for `userSchema`, so why this is getting impacted ?